### PR TITLE
WIP: Status all games

### PR DESCRIPTION
**Pull request by paradoja**

### CODE REVIEW

**Comment by paradoja** [comment]
This is veeeeeeeeeeeeery far from remotely tested.

File: app/controllers/dominions4bot_controller.rb [hunk 1]
```diff
@@ -35,6 +35,11 @@ def soy!(player_game)
 
 
 
+  def pending_games!
+    games = PlayerGame.where(username: @username).map(&:game)
+    respond_with :message, text: "Hola #{@username}, jueguas en #{games.join(", ")}." # no muy útil aún
```
Comment (paradoja) [inline app/controllers/dominions4bot_controller.rb:40]: Juegas

**Comment by g0ldanel** [comment]
is this still a valid `PR` @paradoja  or should we close it?

File: app/controllers/dominions4bot_controller.rb (hunk 1 above)
Comment (g0ldanel) [reply app/controllers/dominions4bot_controller.rb]: this wouldn't be valid in the current schema as the bot has no means to know wht games are still on

**Comment by paradoja** [comment]
Closing as the missing functionality should be added after the changes in #15.

**Pull request closed without merging**